docs/source/json-api/index.rst

```diff
@@ -892,27 +924,41 @@ Some notes on behavior:
    results if you walk the array forwards, backwards, or in random
    order.
 
-Fetch All Known Parties
-=======================
+Fetch by Key Contracts Stream
+-----------------------------
 
-- URL: ``/parties``
-- Method: ``GET``
-- Content: <EMPTY>
+- URL: ``/stream/fetch``
+- Scheme: ``ws``
+- Protocol: ``WebSocket``
 
-HTTP Response
--------------
+List currently active contracts that match one of the given ``templateId - key`` pairs, with continuous updates.
 
-- Content-Type: ``application/json``
-- Content:
+Similarly to `Contracts Query Stream`_, two subprotocols must be passed, as described in `Choosing a party
+<#choosing-a-party>`__.
+
+``application/json`` body must be sent first, formatted according to the following rule:
+
+.. code-block:: none
+
+    [
+        {"templateId": "<template ID 1>", "key": <key 1>}
+        , {"templateId": "<template ID 2>", "key": <key 2>}
+        ...
+        , {"templateId": "<template ID N>", "key": <key N>}
+    ]
```

Comment:
What happens if the list is empty? Do we also allow the format `{"templateId": ..., "key": ...}` if the list contains only one element, i.e., `N = 1`? We should try to be consistent with the other endpoints in this regard. If we're not, I'm fine with addressing this in a separate PR.

Reply:
@hurryabit 

1. Empty list would cause an error: `result: Vector(TextMessage.Strict({"error":"Cannot resolve any templateId from request: List(), unresolved templateIds: Set()"}))`
2. The request must be a list of pairs. If all templates fails resolution, we return error.

I will create a ticket to address the consistency.

Reply:
#4417 created to validate and address the consistency.